cmd/diff.go

```diff
@@ -21,6 +21,14 @@ It will load entities form Kong and then perform a diff on those with
 the entities present in files locally. This allows you to see the entities
 that will be created or updated or deleted.
 `,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) > 0 {
+			return errors.New("diff command cannot take any positional arguments. " +
+				"Try using a flag instead.\n" +
+				"For usage information: deck diff -h")
+		}
```

Comment:
Let's not repeat here and provide the same error message for all commands.
The Go way to do this would be to create a function definition inside the cmd package and then reference it in all places.

Reply:
@hbagdi thanks for the feedback. I have refactored args validation into a `validateNoArgs` func in `cmd/common.go` and now referencing that func in all commands.
The common error message is: `Error: This command cannot take any positional arguments. Try using a flag instead.
For usage information: deck <command> -h`